### DISCUSSION TITLE
About the dangerous settings that will make your device brick.

### DISCUSSION THREAD

**Discussion by CelvestianNesy** [DavidS95/Smokeless_UMAF#15] [Q&A]
Question, will my AMD 5 5600G system brick if I change the curve optimiser settings? When I look into the UMAF bios there are already values that are set based out of Ryzen master, can I ask if my CPU is safe to change curve optimiser? or do I need to use UXTU?

**Comment by DavidS95** [comment]
You are probably safe to use Curve optimizer on your 5600g. These dangerous setting recommendations are nearly exclusively for locked ryzen mobile apus. They as far as i have seen do not effect unlocked ones. So have fun with it.

**Reply by CelvestianNesy** [reply]
Thank you so much, I am not exactly very well versed about this term "APU", can you try to rewrite the dangerous setting part to:

"Warning, using curveoptimiser on locked CPU's is going to brick your device - " (along those lines) for users who don't typically know the terms APU hehe. I'm sorry, it's just a tiny request of clearification. Or maybe something like "You are fine if you are using unlocked desktop CPU's." etc.

However, the p0state vid is something that is dangerous for all devices?

**Reply by DavidS95** [reply]
i am not sure if p0state is dangerous for everyone, everyone who has tried it has either had it do nothing like me or it bricks there device but this tool is used primarily by laptop and handheld users rather than for dyi desktop. No one has tested it and had it work that i know of. If you want to try it be my guest but i wouldnt if i were you. APU is different to cpu because the apus for amd arent chiplit, and have igpus. The 5600g is zen3 just like the 5600x and both have 6 cores, but they dont perform the same and they are decently different. Saying CPU would be wrong because amd's monolithic apus and amd's chiplit cpus are different. All APUs are CPUs but not all CPUs are APUs. I did add a disclaimer about diy parts vs the laptop versions of the same chips.

**Reply by CelvestianNesy** [reply]
I did make a huge bet and tried changing the curveoptimser setting, the PC did not boot, BUT, reseting the CMOS reverted it and everything was fine. It is probabaly for the best to keep that setting "dangerous", if you know what I mean.

I now know not to change that setting anymore. However, your UMAF does indeed work for desktops despite it's limited functionality or weirdness. It could just be me. I'm currently making a video tutorial on how to access hidden menus, for AMD users, which i assume this is what it's directed for.

Thank you so much for making my life easier, and everyone elses. you may not be the orignal creator, i thank her/him too. You are doing us prebuilt gaming users a favour.

**Reply by ff66theone** [reply]
To the curve optimizer also add the RAM speed setting. On my PC, the RAM chip could handle 6000 MT/s just fine, but the poorly built AMD CPU just didn't stand a chance without voltage changes and I had to reset the CMOS.
